### ISSUE TITLE
Could you please share the loss curve?

### ISSUE THREAD

**Issue by venom12138** [hkchengrex/XMem#26]
Hi, could you please share the training loss curve?  I want to see the decline rates and convergence values of dice_loss and ce_loss. Thanks!

**Comment by hkchengrex** [comment]
I don't have it right now. One thing to note is that CE loss might decrease->increase->decrease over the entire training process due to bootstrapped cross entropy.

**Comment by venom12138** [comment]
Thanks for your reply! When I train XMem on my own dataset, the dice loss decreases vey slowly, which is about 0.85 after 10K iterations, while the cross entropy loss has decreased to about 0.02. I wonder if this situation is right.

**Comment by hkchengrex** [comment]
I find the visualizations (images in tensorboard) more helpful. After 10K iterations, there should be meaningful segmentation.

**Comment by MrHsj** [comment]
Thank you for your reply！The default stage 2's iteration is 100k. Do you mean that 10K is actually enough?

**Comment by 1334233852** [comment]
> Thanks for your reply! When I train XMem on my own dataset, the dice loss decreases vey slowly, which is about 0.85 after 10K iterations, while the cross entropy loss has decreased to about 0.02. I wonder if this situation is right.

May I ask for your CE_ Has the loss also decreased first, then increased, and then decreased again to reach a local optimal solution? Have you adjusted the top_ Is the p-value correct? Has the startarm and endwarm been adjusted? May I ask about the approximate size of your dataset? Thank you very much!
